ranger/container/settings.py

```diff
@@ -36,6 +36,8 @@
     'colorscheme': str,
     'column_ratios': (tuple, list),
     'confirm_on_delete': str,
+    'date_format': str,
+    'date_plus_time_format': str,
```

Comment:
Let's rename to `date_and_time_format`, plus kinda sounds like it might be a timezone thing.